contracts/interfaces/ILaunchpegFactory.sol

```diff
@@ -28,9 +36,11 @@ interface ILaunchpegFactory {
         uint256 maxBatchSize,
         uint256 collectionSize,
         uint256 amountForDevs,
-        uint256 batchRevealSize,
         uint256 salePrice,
-        uint256 mintlistPrice
+        uint256 mintlistPrice,
```

Comment:
Move all events, structs and enums to LaunchpegFactory.sol

Reply:
How can I move structs and enums and still use them in the interface ? They are needed because some functions use them as args or returns